aten/src/ATen/native/cuda/Normalization.cu

```diff
@@ -332,14 +335,16 @@ __global__ void batch_norm_backward_kernel(
 
 // TensorAccessor in which the last dimensions are collapsed or expanded as needed
```

Comment:
This function currently does much more than just this comment. I hope that this can be extended to elaborate more on the design choices, i.e., to include when & why to swap dims.